README.md

```diff
@@ -21,42 +21,36 @@ Please press **&#9733; Star** button if you find this library useful, aslo you c
 * Compatible with PHP >= 5.6
 
 ## Installation
-Install the latest version with [Composer](https://getcomposer.org/)
+Install the latest version with [Composer](https://getcomposer.org/) by running
 
 ```bash
 $ composer require maxbanton/cwh:^1.0
 ```
 
 ## Upgrade
-Change in your composer.json
-```
-{
-  "require": {
-    "maxbanton/cwh": "^0.0.3"
-  }
-}
-```
-to
-```
-{
-  "require": {
-    "maxbanton/cwh": "^1.0"
-  }
-}
+Upgrade to the lastest version with [Composer](https://getcomposer.org/) by running
+
 ```
-then run
-```bash
-$ composer update
+$ composer require maxbanton/cwh:^1.0 --update-with-dependencies
 ```
-and change your code
+
+and change your code from
+
 ```php
 <?php
+
+use Maxbanton\Cwh\Handler\CloudWatch;
```

Comment:
Probably a good idea to import `Cloudwatch` here, otherwise PhpStorm (using the Markdown Navigator plugin ) also complains:

![screen shot 2018-04-18 at 08 03 48](https://user-images.githubusercontent.com/605483/38914288-0fce0b10-42df-11e8-954f-15b521f42288.png)